docs/getting-started/overview.md

```diff
@@ -0,0 +1,49 @@
+# Overview
+
+You may be interested in using MCAP for a variety of scenarios throughout your robotics development workflows.
+
+## Convert to MCAP
+
+If you already have existing ROS data that is not in the MCAP file format, you may want to **convert it into MCAP files**.
+
+## Read and write MCAP
+
+If you're starting from scratch, you may want to write code to **read and write your own MCAP data**.
+
+We provide MCAP readers and writers in the following languages, for any of our supported [data serialization formats](../home/data-serialization-formats.md):
+
+- [Python](https://github.com/foxglove/mcap/tree/main/python)
+- [C++](https://github.com/foxglove/mcap/tree/main/cpp)
+- [Go](https://github.com/foxglove/mcap/tree/main/go)
+- [Swift](https://github.com/foxglove/mcap/tree/main/swift)
+- [TypeScript](https://github.com/foxglove/mcap/tree/main/typescript)
+
+## Inspect and visualize MCAP
+
+Once you have MCAP data to work with, you may want to **use the mcap CLI to inspect and interact with your MCAP files**.
```

Comment:
I can't find any explanation of how to install the CLI tool on the main page. All I found was `More Resources > mcap CLI` which takes me away from the documentation site onto GitHub.